### PR TITLE
fixed packer elasticsearch setup

### PR DESCRIPTION
**Pull request by egorksv**

### CODE REVIEW

File: packer/install-elasticsearch7.sh [hunk 1]
```diff
@@ -14,4 +14,12 @@ else
     apt-get install elasticsearch=$ES_VERSION
 fi
 
+mkdir /usr/share/elasticsearch/logs
+mkdir /usr/share/elasticsearch/data
```
Comment (synhershko) [inline packer/install-elasticsearch7.sh:18]: `mkdir -p` no?

File: packer/install-elasticsearch7.sh [hunk 2]
```diff
@@ -14,4 +14,12 @@ else
...
+chown elasticsearch:elasticsearch /usr/share/elasticsearch/logs
+chown elasticsearch:elasticsearch /usr/share/elasticsearch/data
+
 mv elasticsearch.yml /etc/elasticsearch/elasticsearch.yml
+
+sudo /bin/systemctl daemon-reload
+sudo /bin/systemctl enable elasticsearch.service
```
Comment (synhershko) [inline packer/install-elasticsearch7.sh:25]: we are not enabling the es service on purpose - we do that in the userdata script

File: packer/install-elasticsearch7.sh (hunk 1 above)
Comment (egorksv) [reply packer/install-elasticsearch7.sh:18]: /usr/share/elasticsearch/ already exists at this point after installing es, `-p` does not make much difference

File: packer/install-elasticsearch7.sh (hunk 2 above)
Comment (egorksv) [reply packer/install-elasticsearch7.sh:25]: Oh ok, I'll check userdata script - ES was not coming up at all.

**Pull request merged**